src/network/connectionManagerEvents.ts

```diff
@@ -129,7 +129,14 @@ export const createOnEngineConnectionOpened = ({
       label: 'onEngineConnectionOpened',
       message: 'restoreRemoteCameraStateAndTriggerSync',
     })
-    await sceneInfra.camControls.restoreRemoteCameraStateAndTriggerSync()
+
```

Comment:
@jacebrowning This appears to have fixed the gizmo?

So this block of code is the block of code in the system that will ask the engine to send its camera parameters back to the one location in camera controls which then updates the gizmo.

If we do not call this the gizmo never gets updated.

The function `restoreRemoteCameraStateAndTriggerSync` had two workflows, one for setting the camera and one for calling this gizmo workflow.

Since I deleted this function for the `triggerSync` it bork. I should have only deleted the `restoreRemoteCamera` state. 

The `restoreRemoteCamera` state workflow is now moved later in the workflow to a better location for it is only called once instead of twice which would lead to two camera updates and a jarring visual artifact.